dashboard/settings.py

```diff
@@ -85,6 +88,7 @@
         'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
```

Comment:
Might it be better to declare DATABASES as an empty dictionary so we don't accidentally end up with an SQLite3 database, should dj_database_url.config() fail for some reason?